templates/prometheus.yaml

```diff
@@ -105,11 +105,6 @@ spec:
                 targetPort: 5601
                 interval: 30s
         prometheusSpec:
-          image:
-            tag: v2.9.2
```

Comment:
We pinned to an older version of Prometheus since we didn't like the new UI introduced past this version. We were hoping to upgrade to the new UI once [this enhancement](https://github.com/prometheus/prometheus/pull/5758) was merged which makes the UI more palatable - looks like it was released in v2.14.0, but the chart is on v2.13.1. What do you think about using [v2.14.0](https://github.com/prometheus/prometheus/releases/tag/v2.14.0)? It was released on 11/11, so it's about a month old. I try this version out and see if anything obvious breaks

Reply:
I would vote for us moving to v2.14.0 if not obvious breaks are found :)

Reply:
👍 I'll set this to v2.14.0 and play with it.